examples/with-apollo-neo4j-graphql/package.json

```diff
@@ -6,12 +6,13 @@
     "start": "next start"
   },
   "dependencies": {
-    "@apollo/client": "^3.2.7",
-    "apollo-server-micro": "^2.19.0",
+    "@apollo/client": "^3.7.3",
+    "@apollo/server": "4.3.0",
```

Comment:
Does these need to be non-caret dependencies?

Reply:
Great point. Updated!